src/routes/user/registerModel.ts

```diff
@@ -0,0 +1,30 @@
+import * as mongoose from "mongoose";
+
+const formSchema = new mongoose.Schema({
+    login: {
+        required: true,
+        type: String
+    },
+    name: {
+        required: true,
+        type: String
+    },
+    lastName: {
+        required: false,
+        type: String
+    },
+    password: {
+        required: true,
+        type: String
+    },
+    gender: {
+        required: true,
+        type: String
+    },
+    birthDate: {
+        required: true,
+        type: String
+    }
+})
+
+module.exports = mongoose.model('formData', formSchema)

```

Comment:
nie nazywaj rzeczy od tego co akurat robisz xD jakbyś wysyłał inny formularz to dałbyś nazwę `formData2`? Jeżeli to model użytkownika to daj nazwę np `user`